### PR TITLE
Update User Guide with changes from re-modelling

### PR DESCRIPTION
**Pull request by peaol**
- Replace screenshots of our old Ui with the new ones
- Reorganise commands based on its page (Course, Group, Session, Student, etc)
- Update UG with the newly added commands
- Add navigation guide diagram for better clarity

==Ready to Merge==

`Note` still need to update UG again after: 
- fixing broken commands such as find, view
- adding Task and its features (Handled in #135)
- adding screenshots of new gui with session and task tab

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -34,93 +34,246 @@ CLIpboard is optimized **for use via a Command Line Interface** (CLI) while stil
...
+
+
+### Adding a student: `add student`
 
 Adds a student to the student roster.
 
 Format:
 ```
-add n/<NAME> p/<PHONE_NUMBER> e/<EMAIL> sid/<STUDENT_ID> m/<MODULES> [t/<TAG>]
+add n/<NAME> p/<PHONE_NUMBER> e/<EMAIL> sid/<STUDENT_ID> [t/<TAG>]
```
Comment (tyx021) [inline docs/UserGuide.md:276]: Wrong command
```suggestion
add student n/<NAME> p/<PHONE_NUMBER> e/<EMAIL> sid/<STUDENT_ID>
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -34,93 +34,246 @@ CLIpboard is optimized **for use via a Command Line Interface** (CLI) while stil
...
 ```
 
 - `t/<TAG>` field is optional.
```
Comment (tyx021) [inline docs/UserGuide.md:279]: tag is removed
```suggestion
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -131,27 +284,84 @@ Examples:
 - `add n/John Doe p/98765432 e/johnd@example.com sid/A1234567X m/CS2103T`
 - `add n/Roy Balakrishnan p/92624417 e/royb@example.com sid/A7356561X m/CS2105 t/T06`
```
Comment (tyx021) [inline docs/UserGuide.md:285]:
```suggestion
- `add student n/John Doe p/98765432 e/johnd@example.com sid/A1234567X`
- `add student n/Roy Balakrishnan p/92624417 e/royb@example.com sid/A7356561X`
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -131,27 +284,84 @@ Examples:
...
+
+
+### Editing a student: `edit student`
 
 Edits an existing student in the student roster.
 
 Format:
 ```
-edit <INDEX> [n/<NAME>] [p/<PHONE_NUMBER>] [e/<EMAIL>] [sid/<STUDENT_NUMBER>] [m/<MODULES>] [t/<TAG>]
+edit <INDEX> [n/<NAME>] [p/<PHONE_NUMBER>] [e/<EMAIL>] [sid/<STUDENT_NUMBER>] [t/<TAG>]
```
Comment (tyx021) [inline docs/UserGuide.md:309]:
```suggestion
edit <INDEX> [n/<NAME>] [p/<PHONE_NUMBER>] [e/<EMAIL>] [sid/<STUDENT_NUMBER>]
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -131,27 +284,84 @@ Examples:
...
 ```
 
 - Edits student at index specified in `<INDEX>`.
-- At least one field `[n/<NAME>]`, `[p/<PHONE_NUMBER>]`, `[e/<EMAIL>]`, `[sid/<STUDENT_NUMBER>]`, `[m/<MODULES>]`, or `[t/<TAG>]` must be provided.
+- At least one field `[n/<NAME>]`, `[p/<PHONE_NUMBER>]`, `[e/<EMAIL>]`, `[sid/<STUDENT_NUMBER>]`, or `[t/<TAG>]` must be provided.
```
Comment (tyx021) [inline docs/UserGuide.md:313]:
```suggestion
- At least one field `[n/<NAME>]`, `[p/<PHONE_NUMBER>]`, `[e/<EMAIL>]` or `[sid/<STUDENT_NUMBER>]` must be provided.
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -131,27 +284,84 @@ Examples:
...
+
+Copies a student's email to your clipboard.
+
+Format:
+```
+copy <INDEX>
+```
+
+Examples:
+- `copy 1` will copy the email of the 1st student listed to your clipboard.
```
Comment (tyx021) [inline docs/UserGuide.md:333]:
```suggestion
- `copy 1` will copy the email of the 1st student in the list to your clipboard.
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -131,27 +284,84 @@ Examples:
...
+
+Examples:
+- `find John` returns John.
+- `find Alex Yu` returns `Alex Yeoh`, `Bernice Yu` 
+
+
+
+### View a student's information: `view`
+
+Views personal information for a particular student.
```
Comment (tyx021) [inline docs/UserGuide.md:354]:
```suggestion
### Displays a student's information: `select`

Select a particular student and display his / her particulars.
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -131,27 +284,84 @@ Examples:
...
+
+Format:
+```
+view <INDEX>
```
Comment (tyx021) [inline docs/UserGuide.md:358]:
```suggestion
select <INDEX>
```

File: docs/UserGuide.md [hunk 9]
```diff
@@ -131,27 +284,84 @@ Examples:
...
+```
+
+Examples:
+- `view 2` will display the full student details of the 2nd student in the student list on the view panel.
```
Comment (tyx021) [inline docs/UserGuide.md:362]:
```suggestion
- `select 2` will select the 2nd student in the student list and display his / her particulars on the view panel.
```

File: docs/UserGuide.md [hunk 10]
```diff
@@ -169,90 +379,239 @@ Format for deleting a remark:
...
+Format:
+```
+edit session <INDEX> <NEW SESSION NAME>
+```
+
+- Edits session at index specified in `<SESSION>`.
+
+
+Examples:
+- `edit session 1 Tutorial0` will replace the session name of the first session listed to 'Tutorial0'.
```
Comment (tyx021) [inline docs/UserGuide.md:478]:
```suggestion
- `edit session 1 Tutorial1` will rename the first session listed to 'Tutorial1'.
```

File: docs/UserGuide.md [hunk 11]
```diff
@@ -169,90 +379,239 @@ Format for deleting a remark:
...
 ```
+- Selects session at specified index and displays list of students in that session.
+
+Examples:
+- `select 1` will return a list of students from the 1st session.
 
 --------------------------------------------------------------------------------------------------------------
 
-### Uploading a student's photo: `upload`
+## Students by Session Commands
```
Comment (tyx021) [inline docs/UserGuide.md:497]: Sounds abit weird, maybe change it to the following?
```suggestion
## Students under Session Page Commands
```

File: docs/UserGuide.md [hunk 12]
```diff
@@ -169,90 +379,239 @@ Format for deleting a remark:
...
 
-Uploads a student's photo to be displayed on the student roster.
+### Marking attendance of a selected students: `mark`
+
+Marks selected students' attendance as present.
```
Comment (tyx021) [inline docs/UserGuide.md:501]:
```suggestion
Marks selected students' attendance as present for current session.
```

File: docs/UserGuide.md [hunk 13]
```diff
@@ -169,90 +379,239 @@ Format for deleting a remark:
...
 
 Format:
 ```
-upload <LOCAL_FILE_PATH>
+mark <INDEX>, <INDEX>, ...
```
Comment (tyx021) [inline docs/UserGuide.md:505]:
```suggestion
mark <INDEX>
mark <INDEX>, <INDEX>, ...
```

File: docs/UserGuide.md [hunk 14]
```diff
@@ -169,90 +379,239 @@ Format for deleting a remark:
...
-</div>
 
 Examples:
-- `upload /Users/AlexYeoh/Desktop/A0123456X.png`
+- `mark 1` will mark the 1st student as present.
+- `mark 2,3,4` will mark the 2nd, 3rd and 4th students as present.
+
+
+
+### Unmarkikng attendance of a selected students: `unmark`
```
Comment (tyx021) [inline docs/UserGuide.md:514]: Typo
```suggestion
### Marking selected students as absent: `unmark`
```

File: docs/UserGuide.md [hunk 15]
```diff
@@ -169,90 +379,239 @@ Format for deleting a remark:
...
+Examples:
+- `select 1` will return a list of students from the 1st session.
 
 --------------------------------------------------------------------------------------------------------------
 
-### Uploading a student's photo: `upload`
+## Students by Session Commands
 
-Uploads a student's photo to be displayed on the student roster.
+### Marking attendance of a selected students: `mark`
```
Comment (tyx021) [inline docs/UserGuide.md:499]:
```suggestion
### Marking selected students as present: `mark`
```

File: docs/UserGuide.md [hunk 16]
```diff
@@ -169,90 +379,239 @@ Format for deleting a remark:
...
+
+
+
+### Unmarkikng attendance of a selected students: `unmark`
+
+Marks selected students' attendance as absent.
+
+Format:
+```
+unmark <INDEX>, <INDEX>, ...
```
Comment (tyx021) [inline docs/UserGuide.md:520]:
```suggestion
unmark <INDEX>
unmark <INDEX>, <INDEX>, ...
```

**Review by tyx021 — COMMENTED** [review]
Good job! 😄👍
There are a few parts with mistakes and I made some suggestion on some wording, other parts LGTM! 💯

File: docs/UserGuide.md [hunk 17]
```diff
@@ -169,90 +379,239 @@ Format for deleting a remark:
...
 Examples:
-- `upload /Users/AlexYeoh/Desktop/A0123456X.png`
+- `mark 1` will mark the 1st student as present.
+- `mark 2,3,4` will mark the 2nd, 3rd and 4th students as present.
+
+
+
+### Unmarkikng attendance of a selected students: `unmark`
+
+Marks selected students' attendance as absent.
```
Comment (tyx021) [inline docs/UserGuide.md:516]:
```suggestion
Marks selected students' attendance as absent for current session.
```

File: docs/UserGuide.md [hunk 18]
```diff
@@ -34,93 +34,246 @@ CLIpboard is optimized **for use via a Command Line Interface** (CLI) while stil
...
+    - [Selecting a group to display its Student Page: ](#selecting-a-group--select) `select`
+    - [Displaying sessions of a group: ](#displaying-sessions-of-a-group--session) `session`
+    - [Displaying tasks of a group: ](#displaying-tasks-of-a-group--task) `task`
+- [Students Page Commands: ](#students-page-commands)
+    - [Adding a student: ](#adding-a-student--add-student) `add student`
+    - [Deleting a student: ](#deleting-a-student--delete-student) `delete student`
+    - [Editing a student: ](#editing-a-student--edit-student) `edit student`
+    - [Copying a student's email: ](#copying-a-students-email--copy) `copy`
+    - [Finding students by name: ](#finding-students-by-name--find) `find`
+    - [View a student's information: ](#view-a-students-information--view) `view`
```
Comment (tyx021) [inline docs/UserGuide.md:67]:
```suggestion
    - [View a student's information: ](#view-a-students-information--view) `view`
    - [Displays a student's information:  ](#displays-a-students-information--view) `select`
```

File: docs/UserGuide.md [hunk 19]
```diff
@@ -34,93 +34,246 @@ CLIpboard is optimized **for use via a Command Line Interface** (CLI) while stil
...
+    - [Adding or deleting a remark: ](#adding-or-deleting-a-remark--remark) `remark`
+    - [Upload a student's photo: ](#uploading-a-students-photo--upload) `upload`
+    - [Sorting list of students: ](#sorting-list-of-students--sort) `sort`
+- [Session Page Commands: ](#session-page-commands)
+    - [Adding a session: ](#adding-a-session--add-session) `add session`
+    - [Deleting a session: ](#deleting-a-session--delete-session) `delete session`
+    - [Editing a session: ](#editing-a-session--edit-session) `edit session`
+    - [Selecting a session to start attendance-taking: ](#selecting-a-session-to-start-attendance-taking--select) `select`
+- [Students by Session Commands: ](#students-by-session-commands)
```
Comment (tyx021) [inline docs/UserGuide.md:76]:
```suggestion
- [Students under Session Commands: ](#students-under-session-page-commands)
```

**Comment by IsabelChong** [comment]
Good job @peaol and @tyx021! Have looked through the initial commit and suggestions, LGTM 👍

Task commands will be updated in the UG in a later PR to be updated here.

**Pull request merged**